docs/about-filecoin/ipfs-and-filecoin.md

```diff
@@ -11,32 +11,36 @@ This page aims to explain the relationship between the IPFS and Filecoin project
+IPFS is great for getting started using content addressing for all sorts of distributed web applications. In the majority of these cases: 
+
+- Data is provided by user's own nodes. Otherwise must rely on other peers to voluntarily/altruistically storing data or on a centralized pinning service.
 - Centralized IPFS pinning services must be trusted to do their job. IPFS brings no built-in provisions to verify that data is being stored and correctly provided by the pinning service.
-- IPFS by itself works very well for popular content (with many providers), for organizations where there are incentives to sync and store data in multiple nodes and for situation where strong social contracts can be used to ensure the content remains hosted and maintained long-term.
+- Popular content is more easily accessible. Popular content (with many providers) naturally becomes faster/easier to retrieve in IPFS, which is great when there are external incentives to sync and store data in multiple nodes, and for situation where strong social contracts can be used to ensure the content remains hosted and maintained long-term.
+
+### Data persistence with Filecoin
 
-### Using Filecoin
+Filecoin builds on the content addressing of IPFS to add longer term data persistence via cryptoeconomic incentives. With Filecoin: 
 
 - Clients make _storage deals_ with miners to store data. The network verifies that the miners are correctly storing the data. Small payments are made on a regular basis for the duration of the _storage deal_.
 - Miners that do not honor the storage deal are penalised.
 - Content retrieval might be offered by storage miners directly, or by specialized retrieval miners. The user requesting the data pays for this service.
 - Filecoin excels at storing large amounts of data for long periods of time.
 
-### Using both
+### Using IPFS and Filecoin together
 
-Some solutions combine the best of the two systems, backing up data on the Filecoin network and at the same time providing the data through the IPFS network. This ensures that data is constantly available and can be retrieved quickly, while also making sure that the data is safely backed up on the Filecoin network. [Powergate](../build/powergate.md) is a multitiered file storage API built on Filecoin and IPFS, and an index builder for Filecoin data.
+While using Filecoin does inherently use IPFS under the hood, many solutions also combine the two networks by backing up data on the Filecoin network and actively providing the data through the IPFS network. This ensures that data is constantly available and can be retrieved quickly, while also making sure that the data is safely and verifiably backed up on the Filecoin network over time. [Powergate](../build/powergate.md) is a multitiered file storage API built on Filecoin and IPFS, and an index builder for Filecoin data.
```

Comment:
i think this paragraph is just messy and contributes to misinformation. it maybe should distinguish go-ipfs from IPFS

Reply:
> actively providing the data through the IPFS network.

- "backing up data on Filecoin" means "providing data through IPFS network"
- though it does not mean "using the IPFS DHT for content routing using the provider record system"

Reply:
I actually wouldn't classify "backing up data to filecoin" as "providing data to the IPFS network" until there is a retrieval market/index that is accessible for data discovery - but agree it's rather confusing. Will iterate.

Reply:
```suggestion
Many solutions combine the two networks to get the best of both worlds: IPFS for content addressing & data discovery, and Filecoin for longer-term persistence. To achieve this, many solutions like [Powergate](../build/powergate.md) back up data on the Filecoin network while also ensuring content is discoverable in the IPFS Public DHT. This ensures that data is constantly available and can be retrieved quickly, while also making sure that the data is safely and verifiably backed up on the Filecoin network over time.
```

Reply:
```suggestion
Many solutions combine the two networks to get the best of both worlds: IPFS for content addressing & data discovery, and Filecoin for longer-term persistence. To achieve this, services like [Powergate](../build/powergate.md) back up data on the Filecoin network while also ensuring content is discoverable in the IPFS Public DHT. Data is constantly available and can be retrieved quickly, while also making sure that it is safely and verifiably backed up on the Filecoin network over time.
```